suborbital/sat
# Add Capabilities to Sat

This PR adds Suborbital Compute capabilities to Sat. 

Testing:
1. Run `make docker`
2. [Start](https://docs.suborbital.dev/installing/2.-run-compute-locally) Suborbital Compute, set any capabilities you'd like to test in `config/scc-config.yaml`
3. Run `ops/test-capabilities` This starts a `sat:dev` Docker image in capabilities mode with the [`http-cap` runnable](https://github.com/suborbital/sat/blob/917a23d97606c97a364d672a94b63bbe58aefd48/examples/http-cap/src/lib.rs) running on port :8079
4. Send some requests to the Sat instance!

`curl localhost:8079 -d "https://suborbital.dev"` (should work)
`curl localhost:8079 -d "http://localhost:8082"` (should fail)
`curl localhost:8079 -d "https://1.1.1.1"` (should fail)
